projects/ngx-d3timeline/src/lib/store/reducer.ts

```diff
@@ -49,7 +49,12 @@ export function reducer(state: State, action: Actions): State {
       const activities = state.positionedActivities.map(activity =>
         activity.id === action.payload.id ? action.payload : activity
       );
-      return { ...state, positionedActivities: activities, dragEvent: null };
+      return {
+        ...state,
+        positionedActivities: activities,
+        dragEvent: null,
```

Comment:
possibly you could not have `lastDraggedActivityId` and get the Id from `dragEvent` when it changes from something to null? haven't though it through fully...

seems a little bit questionable having `lastDraggedActivityId` hanging around in the state.

Reply:
> possibly you could not have lastDraggedActivityId and get the Id from dragEvent when it changes from something to null? haven't thought it through fully...

I started doing something like this, it gets tricky.

>seems a little bit questionable having lastDraggedActivityId hanging around in the state.

For me its no different than having selectedId in the store.